shaka/include/shaka/ShakaPlayer.h

```diff
@@ -115,11 +115,16 @@ SHAKA_EXPORT
 SHAKA_EXPORT
 @interface ShakaPlayer : NSObject
 
+/** A client which will receive player events */
+@property (atomic, weak) id<ShakaPlayerClient> client;
```

Comment:
Does this need `_Nullable` for it to be nullable in Swift?  Also put this below the `init` methods.

Reply:
I was planning to add nullability tags in another pull request, because they are needed in a lot of places, and once you put one you are forced to put them all.